src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
+	 *
+	 * @since TBD
+	 */
+	public function enable(  ) {
+		if ( has_action( 'tribe_log', [ $this, 'dispatch_log' ] ) ) {
+			return;
+		}
+
+		remove_action( 'tribe_log', [ $this, 'dispatch_log' ] );
+	}
+
+	/**
+	 * Disables the logging functions.
+	 *
+	 * @since TBD
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: c10f2ec527c98094521b16e35fa1bd7c</sub>